src/components/FolderPage/CardListItem/CardListItem.js

```diff
@@ -3,31 +3,110 @@ import { getTimeAgo, formatDate } from "util/time";
 import noImg from "assets/noImg.png";
 import starImg from "assets/Star 1.svg";
 import kebabImg from "assets/kebab.svg";
+import { useEffect, useRef, useState } from "react";
+
+function CardListItem({ link, setModal, setLink }) {
+  const [btnclicked, setBtnClicked] = useState(false);
```

Comment:
버튼을 클릭했냐 안했냐보다 모달이 열리는 상태냐 아니냐가 더 쉽게 파악이 가능 할것 같아요?!
```suggestion
  const [isKebabMenuOpen, setIsKebabMenuOpen] = useState(false);
```